api/services/google-tasks.ts

```
@@ -113,7 +114,13 @@ function extractMetadataFromNotes(notes: string): {
 
     switch (key) {
       case 'Priority':
-        metadata.priority = value === 'High';
+        const priorityMap: Record<string, number> = {
+          'None': 0,
+          'Low': 1,
+          'Medium': 2,
+          'High': 3
+        };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)